aiopg/connection.py

```diff
@@ -421,3 +420,27 @@ def __del__(self):
     def notifies(self):
         """Return notification queue."""
         return self._notifies
+
+
+class _CursorContextManager:
```

Comment:
See https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/client.py#L410-L467 for proper context manager implementation